packages/pwa-devdocs/src/technologies/basic-concepts/app-shell/index.md

```diff
@@ -1,12 +1,17 @@
 ---
 title: Application Shell
+contributors:
+    - gavin2point0
 ---
 
-The Magento PWA Studio uses application shell architecture to quickly load pages on mobile apps.
+Magento PWA Studio uses an application shell architecture to shorten the time it takes to load pages.
```

Comment:
> to shorten the time it takes to load pages.

We may want to change this to instead say that it shortens the amount of time it takes to load some branded experience in the UI rather than a white page, since the useful content for the page technically hasn't loaded or executed yet.